backend/supabase/wrapped-functions.sql

```diff
@@ -0,0 +1,42 @@
+
+create
+or replace function get_monthly_bet_count_and_amount (user_id_input text) returns table (
+  month TIMESTAMPTZ,
+  bet_count bigint,
+  total_amount numeric
+) as $$
+BEGIN
+    RETURN QUERY
+    SELECT
+        date_trunc('month', created_time) AS month,
+        COUNT(bet_id) AS bet_count,
+        SUM(amount) AS total_amount  -- Add this line to calculate the sum of amounts
```

Comment:
This would discount negative bet amounts, mainly sales. If you only had a large sale at a profit, you could even have a negative total bet amount, which doesn't really make sense lol.

I would add an absolute value: SUM(ABS(amount)). But then you need to filter out redemptions below.